### DISCUSSION TITLE
updated /src/class/hid/hid.h with digitizer enum - how is the process to merge this stuff into the tinyusb stack

### DISCUSSION THREAD

**Discussion by Commotari** [hathach/tinyusb#2144] [General]
Hello guys,

on my project i have to implement a resistive touch screen (already done) and bring the x/y/touched data over USB - compatible to an old project (done on a TI microcontroller).
for this i need a descriptor with HID_USAGE_PAGE_DIGITIZER.
the necessary defines for the USAGE are not in hid.h
i updated the file by adding table 18: digitizer page to hid.h

==> how is the process to merge that into the next tinyusb stack version?
(same question probably applies to my changes to the STM32 device layer where i added stuff to make it work on my STM32G0B1 MCU)

grettings and keep up the good work,
  WhyLee (Stefan Weilhartner)

**Comment by hathach** [comment]
you can make an PR to update hid.h , if you are not familliar with PR, github has a guide for how to https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request-from-github-desktop
